### PR TITLE
Add tooltips to gadget search buttons

### PR DESCRIPTION
**Pull request by jeffsieu**
Resolves #37

### CODE REVIEW

File: ropa/gui/gui.py [hunk 1]
```diff
@@ -95,6 +95,13 @@ def _load_buttons(self):
         self._bind_button_clicked(self.search_semantics_button,
                                   controller.semantics_function)
 
+        self.search_instructions_button.setToolTip(
+            "Search for instruction blocks")
+        self.search_poppopret_button.setToolTip(
+            "Search for blocks containing <i>POP POP RET</i> sequences")
+        self.search_semantics_button.setToolTip(
+            "Search for blocks using Ropper's semantics")
```
Comment (daniellimws) [inline ropa/gui/gui.py]: s/semantics/semantic searching function/

File: ropa/gui/gui.py [hunk 2]
```diff
@@ -95,6 +95,13 @@ def _load_buttons(self):
         self._bind_button_clicked(self.search_semantics_button,
                                   controller.semantics_function)
 
+        self.search_instructions_button.setToolTip(
+            "Search for instruction blocks")
```
Comment (daniellimws) [inline ropa/gui/gui.py]: s/instruction blocks/gadgets/

File: ropa/gui/gui.py (hunk 1 above)
Comment (daniellimws) [reply ropa/gui/gui.py]: s/blocks/gadgets/

File: ropa/gui/gui.py [hunk 3]
```diff
@@ -95,6 +95,13 @@ def _load_buttons(self):
...
+        self.search_poppopret_button.setToolTip(
+            "Search for blocks containing <i>POP POP RET</i> sequences")
```
Comment (daniellimws) [inline ropa/gui/gui.py]: s/blocks/gadgets/

**Comment by jeffsieu** [comment]
Changes have been made.

**Comment by daniellimws** [comment]
Gut

**Pull request merged**